src/WordSearch.java

```diff
@@ -20,26 +21,40 @@ public static void main(String[] args) {
     /*
     Checking File Extension of the filePath whether '.txt' or '.json' file.
     And Calling processFile method for searching the file in system.
+    If format of file is not supported, Creating an object for WordSearchHelper and calling a method dataBaseStorage from that class.
+    Storing the results in DataBase.
      */
-    public static void fileExtensionCheck(String filePath, String keyWordToSearch) {
-        if (filePath.endsWith(".txt") || filePath.endsWith(".json")) {
+    public static void fileExtensionCheck(String filePath, String keyWordToSearch) throws SQLException {
+        String txt = ".txt";
```

Comment:
These can also be put in Constants class and be accessed from constants class because they are constants

Reply:
Updated.